x/proof/types/claim.go

```diff
@@ -55,18 +55,24 @@ func (claim *Claim) GetClaimeduPOKT(
 		return sdk.Coin{}, err
 	}
 
-	computeUnitsToTokenMultiplierRat := new(big.Rat).SetUint64(sharedParams.GetComputeUnitsToTokensMultiplier())
+	// Calculate the fractional uPOKT amount of a single compute unit.
+	computeUnitsToUpoktMultiplierRat := new(big.Rat).SetFrac64(
+		// CUTTM is a GLOBAL network wide parameter.
+		int64(sharedParams.GetComputeUnitsToTokensMultiplier()),
+		// The uPOKT cost granularity of a single compute unit.
```

Comment:
also a global network wide param?

Reply:
```
  // DEV_NOTE: A global network parameter is used instead of an in-code constant
  // to enable delayed params updates and and prevent affecting settling sessions
  // from using parameters that were not in effect at the time of the session.
```